init-tools.sh

```diff
@@ -47,7 +45,7 @@ if [ ! -e $__PROJECT_JSON_FILE ]; then
     echo "Running: $__scriptpath/init-tools.sh" > $__init_tools_log
     if [ ! -e $__DOTNET_PATH ]; then
         echo "Installing dotnet cli..."
-        __DOTNET_LOCATION="https://dotnetcli.blob.core.windows.net/dotnet/preview/Binaries/${__DOTNET_TOOLS_VERSION}/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz"
+        __DOTNET_LOCATION="https://corefxci.blob.core.windows.net/dotnetcli/${__DOTNET_PKG}.${__DOTNET_TOOLS_VERSION}.tar.gz"
```

Comment:
Until CLI has packages and we can move back to their feeds, yes.

We could also consider just uploading these packages to the dotnetcli blob storage, with names that sort of match the old scheme (replacing -ubuntu with -ubuntu.14.04-x64 ideally to disambiguate between ubuntu versions), if that's preferable (I would need to chat quickly with @piotrpMSFT before doing so, but it might be helpful for them to have them for bootstrapping anyway).